.github/workflows/tii-octomap-server2-coverity.yaml

```diff
@@ -3,20 +3,20 @@ name: tii-octomap-server2-coverity
 on:
   repository_dispatch:
   push:
-    branches:
-      - master
 
 jobs:
   coverity:
     runs-on: octomap-server2-self-hosted-coverity
     steps:
-      - uses: actions/checkout@v3
-      - name: Login to GitHub Container Registry
-        uses: docker/login-action@v2
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Coverity scan 👁️
+        uses: addnab/docker-run-action@v3
         with:
-          registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
-      - name: Run coverity scan
-        run:
-          docker run --rm --entrypoint=/entrypoint-coverity.sh -e PROJECT_NAME=${{ github.event.repository.name }} -v "$(pwd):/main_ws/src"  ghcr.io/tiiuae/fog-ros-baseimage-builder-coverity:main
+          registry: ghcr.io
+          image: ghcr.io/tiiuae/fog-ros-baseimage-builder-coverity:main
+          options: -e PROJECT_NAME=${{ github.event.repository.name }} -v ${{ github.workspace }}:/main_ws/src
+          run: |
+            /entrypoint-coverity.sh
```

Comment:
I would define the entrypoint in the base image (`ENTRYPOINT ["/entrypoint-coverity.sh"]`), so it doesn't need boilerplate in all call sites.

It's a small thing but this implementation will be copy-pasted to very many places.